virt/virt-2-5-release-notes.adoc

```diff
@@ -66,6 +66,8 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 //CNV-7170 - CDI import of container disks is faster and more efficient
 
 //CNV-7168 - DataVolume and storage diagnostics
+It is now easier to diagnose and troubleshoot issues when preparing virtual machine (VM) disks that are managed by DataVolumes. In the VM configuration file, specify `spec:machine:type:dataVolume` or `spec:machine:type:""` for storage that is managed by DataVolumes.
+If you specify any other value for `type`, such as `persistentVolumeClaim`, a warning is displayed and the virtual machine does not start.
 
```

Comment:
We recently changed this behavior in response to a few bugs.  Now, if a PVC is owned by a DataVolume, we check the DataVolume status to ensure the volume is ready for use if the volume type is dataVolume or persistentVolumeClaim.  

Note that in your text above you reference the wrong field.  I believe it should be spec:volumes:type

Reply:
@aglitke What is the expected value of the DataVolume status field? And if it is any other value, is the behavior still similar to above - a warning is displayed and the virtual machine does not start?

Reply:
@aglitke I found this existing documentation on [Diagnosing DataVolumes using events and conditions](https://docs.openshift.com/container-platform/4.5/virt/logging_events_monitoring/virt-diagnosing-datavolumes-using-events-and-conditions.html). 
What are we adding specifically in this release that would make it easier for users to diagnose problems with DataVolumes?

Reply:
* During upload we report more potential errors up-front (https://issues.redhat.com/browse/CNV-5192)
* We generate DV events when conditions are updated which makes it easier to see the progression of a DV through its phases (https://issues.redhat.com/browse/CNV-463)

Reply:
The status should show phase=Succeeded

Reply:
@aglitke I have revised the content based on the new information